connect/cli/plugins/translation/commands.py

```diff
@@ -59,5 +60,53 @@ def cmd_export_translation(config, translation_id, output_file, output_path):
         )
 
 
+@grp_translation.command(
+    name='activate',
+    short_help='Active a translation.',
+)
+@click.argument('translation_id', metavar='TRANSLATION_ID', nargs=1, required=True)
+@click.option(
+    '--force-activate',
```

Comment:
I think it should be just `--force` for all the commands that we have. There is no need to invent different flags for force for different commands. Wdyt?

Reply:
Yes you are right because for primarize command, it should have the same option flag and if I do not generalize I would be in the situation of invent another one.